src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -696,23 +711,75 @@ cbuf_create(spdid_t spdid, unsigned long size, int cbid)
 	ret = (int)id;
 done:
 	tracking_end(NULL, CBUF_CRT);
-	CBUF_RELEASE();
-
 	return ret;
 free:
 	cmap_del(&cbufs, id);
 	free(cbi);
 	goto done;
 }
 
+int
+cbuf_create(spdid_t spdid, unsigned long size, int cbid)
+{
+	int ret = 0;
+	unsigned int id = (unsigned int)cbid;
+
+	printl("cbuf_create\n");
+	CBUF_TAKE();
+	ret = __cbuf_create(spdid, size, cbid, 0);
+	CBUF_RELEASE();
+
+	return ret;
+}
+
+int
+__map_and_meta(spdid_t spdid, int cbid, int size, vaddr_t daddr)
```

Comment:
One of the larger outstanding issues was refactoring this, which I think I just did (not in this PR).